andela/travel_tool_front
# #160725781 Clear Request Details Modal

#### What does this PR do?
clear existing request details on request modal before rendering.

#### Description of Task to be completed?
- Clear request details modal

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login if you haven't
- Navigate to either My Request or My Approval page
- Inspect the page and navigate to the Network menu
- Change the throttling setting to any 3G network (see picture below, throttling setting is label `Offline`)
- Click on any request Details
- The picture below will be displayed first and after which the request details will be loaded on the request modal

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160725781](https://www.pivotaltracker.com/story/show/160725781)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-30 at 6 29 13 pm" src="https://user-images.githubusercontent.com/28366272/46279109-24d0dc00-c560-11e8-91b0-50c47a62b0d4.png">


#### Questions:
N/A
